tools/config-markdown.py

```
@@ -1,207 +1,163 @@
+                    md.append(f"{header_prefix} {sub_item.get('short', sub_item.get('description', ''))}\n")
+                    if sub_item.get('short') and sub_item.get('description') and sub_item.get('short') != sub_item.get('description'):
+                        md.append(f"\n{sub_item.get('description')}\n")
+                    md.extend(insert_images_and_header(sub_item))
+                    if sub_item.get('author'):
+                        md.append(f"**Author:** {sub_item['author']}\n")
+                    if sub_item.get('status'):
+                        md.append(f"**Status:** {sub_item['status']}\n")
+                    first_sub = False
+
+                if sub_item.get('command'):
+                    cmd = sub_item['command'][0] if isinstance(sub_item['command'], list) else sub_item['command']
```

Ruff (0.8.2)

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)